GetStreamActivityFeed/User Story/Root/RootPresenter.swift

```diff
@@ -23,7 +23,7 @@ final class RootPresenter {
             return
         }
         
-        guard let token = Bundle.main.streamToken else {
+        guard let token = Bundle.main.streamValue(for: .streamToken) else {
```

Comment:
I still can't build the project, it says:
> 'streamValue' is inaccessible due to 'private' protection level
whereever `streamValue` is used

Reply:
Fixed. Didn't realize it was in another package. Found the public function I should be using instead :)